crest/Assets/Crest/Crest/Scripts/LodData/OceanDepthCache.cs

```diff
@@ -439,7 +512,8 @@ public bool Validate(OceanRenderer ocean, ValidatedHelper.ShowMessage showMessag
                     (
                         "Depth cache is outdated.",
```

Comment:
Should we auto-populate the cache after a user clicks a fix function which outdates the cache (and is not baked cache)?

Reply:
good shout, done. i assume i should pass `true` to the populate function?

Reply:
Nice. Correct. That way it updates the changed components.